en/flight_controller/omnibus_f4_sd.md

```diff
@@ -0,0 +1,123 @@
+# Omnibus F4 SD
+
+The *Omnibus F4 SD* is a controller board designed for racers. 
+In contrast to a typical racer board it has some additional features, such as an SD card and a faster CPU.
+
+These are the main differences compared to a [Pixracer](../flight_controller/pixracer.md):
+- Lower price
+- Fewer IO ports (though it's still possible to attach a GPS or a Flow sensor for example)
+- Less RAM (192 KB vs. 256 KB) and FLASH (1 MB vs. 2 MB)
+- Same board dimensions as a *Pixracer*, but slightly smaller form factor (because it has less connectors)
+
+> **Tip** All the usual PX4 features can still be used for your racer!
+
+<img src="../../assets/flight_controller/omnibus_f4_sd/board.jpg" width="400px" title="Omnibus F4 SD" />
+
+
+## Key Features
+
+* Main System-on-Chip: [STM32F405RGT6](https://www.st.com/en/microcontrollers/stm32f405rg.html)
+  * CPU: 168 MHz ARM Cortex M4 with single-precision FPU
+  * RAM: 192 KB SRAM
+  * FLASH: 1 MB
+* Standard racer form factor: 36x36 mm with standard 30.5 mm hole pattern
+* MPU6000 Accel / Gyro
+* BMP280 Baro (not all boards have it mounted)
+* microSD (logging and parameters)
+* Futaba S.BUS and S.BUS2 / Spektrum DSM2 and DSMX / Graupner SUMD / PPM input / Yuneec ST24
+* OneShot PWM out (configurable)
+* built-in current sensor
+* built-in OSD chip (AB7456 via SPI, not supported yet)
+
+
+## Where to Buy
+
+The board is produced by different vendors, with some variations (e.g. with or without a barometer).
+If the Betaflight OMNIBUSF4SD target can be used, then it should work with PX4 as well.
```

Comment:
What is the "Betaflight OMNIBUSF4SD target". As a PX4 developer I don't know what this is, and how I would test it.

Reply:
Basically it just means that if you search for 'OMNIBUSF4SD' on the product page and find it, you're good to go.

Reply:
It should probably say that then :-)

Reply:
Right, do you want to do the rewrite?

Reply:
"want" would be overstating it :-). Done in https://github.com/PX4/px4_user_guide/pull/289/commits/532c0042732698521e21037492fe536e19483efb

Reply:
Haha, thanks!